_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -69159,7 +69159,7 @@
 /area/maintenance/port)
 "cCF" = (
 /obj/effect/turf_decal/bot,
-/obj/structure/tank_holder/emergency_oxygen,
```

Comment:
ah, the emergency oxygen tank holder, truly a powerful maintenance item that people meta rush consistently

Reply:
oh, wait, you just swapped this thing's location with a plant, my bad

Reply:
it's replaced by plant, plants are nice